pescador/zmq_stream.py

```
@@ -181,18 +180,19 @@ def generate(self, max_batches=None):
             worker = mp.Process(target=SafeFunction(zmq_worker),
                                 args=[port, self.streamer, terminate],
                                 kwargs=dict(copy=self.copy,
-                                            max_batches=max_batches))
+                                            max_iter=max_iter))
 
             worker.daemon = True
             worker.start()
 
             # Yield from the queue as long as it's open
             while True:
-                yield zmq_recv_batch(socket)
+                yield zmq_recv_data(socket)
 
         except StopIteration:
             pass
 
+        # TODO: Fix bare except
```

no, this is an intentional bare except that gets reraised.  IIRC, this is here because of some weirdness with transportable exceptions.